modules/file/file.go

```diff
@@ -103,6 +105,7 @@ type options struct {
 	MatchLimit   float64  `json:"matchlimit"`
 	Debug        string   `json:"debug,omitempty"`
 	ReturnSHA256 bool     `json:"returnsha256,omitempty"`
+	Decompress   bool   `json:"decompress,omitempty"`
```

Comment:
You should `go fmt` your code before sending PRs. If you're a vim user, I strongly recommend you use [vim-go](https://github.com/fatih/vim-go) which does all that for you automatically.